content/news/sdsc_award_arribas-bel.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Dani Arribas-Bel, SDSC Award
+date: 2020-10-23
+description:  summary: Congratulations to Dani Arribas-Bel for being named Spatial Data Scientist of the Year
```

Comment:
should this have the `summary:`? 

no worries if intentional, just checking

Reply:
no it shouldn't, I miss that when i merged it.